### PR TITLE
add initial DWH PR template

### PR DESCRIPTION
**Pull request by andrewq**
Discuss, but **DO NOT MERGE**. This will take effect immediately in all repos; let's make sure it's what we want.

### CODE REVIEW

**Comment by farhanalam** [comment]
What does this do?

File: pull_request_template.md [hunk 1]
```diff
@@ -1,2 +1,14 @@
+_Describe the contents of your PR, how to manually test it, and provide mockups that help provide context._
 
-
+- [ ] I have manually tested the code
+- [ ] The PR name contains the ticket number
```
Comment (araney) [inline pull_request_template.md:4]: Which ticket number? User story? Task? Both?

I prefer both.

File: pull_request_template.md [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+- [ ] All of my commits contain the ticket number
+- [ ] My code follows the style guidelines of this project
+- [ ] I have performed a self-review of my own code
+- [ ] I have made corresponding changes to the documentation
```
Comment (araney) [inline pull_request_template.md:8]: Does this imply creating documentation where there was none?

**Comment by andrewq** [comment]
This will insert itself verbatim into every opened PR description.

File: pull_request_template.md [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+- [ ] My changes generate no new warnings
+- [ ] All automated tests pass
+- [ ] I have added tests that prove my fix is effective or that my feature works
```
Comment (araney) [inline pull_request_template.md:11]: Does this include creating front end tests in projects that don’t yet have them?

**Comment by andrewq** [comment]
We've taken discussion offline.

**Pull request closed without merging**